WalletWasabi.Fluent/Views/Login/LoginView.axaml

```diff
@@ -70,7 +70,7 @@
           </DockPanel>
 
           <!-- Hardware & Watch only wallet -->
-          <Button Content="Login" IsVisible="{Binding !IsPasswordNeeded}" HorizontalAlignment="Center" Classes="action" Command="{Binding NextCommand}"/>
+          <Button Content="Login" IsVisible="{Binding !IsPasswordNeeded}" HorizontalAlignment="Center" Classes="action" Command="{Binding NextCommand}" />
```

Comment:
Rename `Login` here as well. 
The layout is different for HW wallet since there is no password. That is why there is two login button.
I guess this view could be simplified to have only one button, but for now, just rename it.